### PR TITLE
Added Flight School

### PR DESCRIPTION
**Pull request by Vega-Punk**

### CODE REVIEW

File: WasabiWords.lua [hunk 1]
```diff
@@ -609,7 +664,16 @@ wasabi_words:add_button("ShinyWasabi", function() --Original script by ShinyWasa
...
+        stats.set_int('MPX_LOWRIDER_FLOW_COMPLETE', 3)
+        stats.set_int('MPX_LOW_FLOW_CURRENT_PROG', 9)
+        stats.set_int('MPX_LOW_FLOW_CURRENT_CALL', 9)
+        stats.set_bool('MPX_HAS_WATCHED_BENNY_CUTSCE', true)
+        stats.set_bool('MPX_LOW_FLOW_CS_DRV_SEEN', true)
+        stats.set_bool('MPX_LOW_FLOW_CS_TRA_SEEN', true)
+        stats.set_bool('MPX_LOW_FLOW_CS_FUN_SEEN', true)
+        stats.set_bool('MPX_LOW_FLOW_CS_PHO_SEEN', true)
+        stats.set_bool('MPX_LOW_FLOW_CS_FIN_SEEN', true)
+        stats.set_bool('MPX_LOW_BEN_INTRO_CS_SEEN', true)
```
Comment (gir489returns) [inline WasabiWords.lua:676]: What is this for? I don't see these checked in the scripts.

They are bool stats 174 -> 180. Remove them if not required.

File: WasabiWords.lua [hunk 2]
```diff
@@ -609,7 +664,16 @@ wasabi_words:add_button("ShinyWasabi", function() --Original script by ShinyWasa
         stats.set_int("MPX_SALV23_GEN_BS", -1) -- polgauntlet trade price
         stats.set_int("MPX_SALV23_SCOPE_BS", -1) -- police5 trade price
         stats.set_int("MPX_MOST_TIME_ON_3_PLUS_STARS", 300000) -- police4 trade price
-        stats.set_int("MPX_LOWRIDER_FLOW_COMPLETE", 1)
+        stats.set_int('MPX_LOWRIDER_FLOW_COMPLETE', 3)
+        stats.set_int('MPX_LOW_FLOW_CURRENT_PROG', 9)
+        stats.set_int('MPX_LOW_FLOW_CURRENT_CALL', 9)
```
Comment (gir489returns) [inline WasabiWords.lua:669]: These control the user's progress through the low rider missions. Setting them to 9 will keep the user forever stuck at the end. Remove this.

File: WasabiWords.lua [hunk 3]
```diff
@@ -161,18 +160,16 @@ wasabi_words:add_button("ShinyWasabi", function() --Original script by ShinyWasa
...
-        unlock_packed_bools(42068, 42069) --Snowman Finish for Combat Pistol, Santa's Helper Finish
+        unlock_packed_bools(42068, 42069) --Snowman Finish, Santa's Helper Finish
         unlock_packed_bools(42111, 42111) --The Coast Guard
         unlock_packed_bools(42119, 42123) --Yeti Outfit, Snowman Finish, Santa's Helper Finish, Skull Santa Finish, riot unlocked
         unlock_packed_bools(42125, 42125) --riot trade price
-        unlock_packed_bools(42128, 42129) --eCola Festive Sweater, Sprunk Festive Sweater
-        unlock_packed_bools(42130, 42146) --1 Party Hat, 2 Party Hat, 3 Party Hat, 4 Party Hat, 5 Party Hat, 6 Party Hat, 7 Party Hat, 8 Party Hat, 9 Party Hat, 10 Party Hat, 11 Party Hat, 12 Party Hat, 13 Party Hat, 14 Party Hat, 15 Party Hat, Bronze Party Outfit, Silver Party Outfit
+        unlock_packed_bools(42128, 42146) --eCola Festive Sweater, Sprunk Festive Sweater, 1 Party Hat, 2 Party Hat, 3 Party Hat, 4 Party Hat, 5 Party Hat, 6 Party Hat, 7 Party Hat, 8 Party Hat, 9 Party Hat, 10 Party Hat, 11 Party Hat, 12 Party Hat, 13 Party Hat, 14 Party Hat, 15 Party Hat, Bronze Party Outfit, Silver Party Outfit
```
Comment (gir489returns) [inline WasabiWords.lua:172]: No idea what you changed here, revert this.

**Review by gir489returns — CHANGES_REQUESTED** [review]

File: WasabiWords.lua (hunk 3 above)
Comment (gir489returns) [reply WasabiWords.lua:172]: lmao nevermind

File: WasabiWords.lua [hunk 4]
```diff
@@ -161,18 +160,16 @@ wasabi_words:add_button("ShinyWasabi", function() --Original script by ShinyWasa
...
         unlock_packed_bools(41994, 41994) --Junk Energy Racing Suit
         unlock_packed_bools(41996, 41996) --??? T-Shirt
         unlock_packed_bools(42054, 42054) --Strapz Bandana
         unlock_packed_bools(42063, 42063) --The LS Panic
-        unlock_packed_bools(42068, 42069) --Snowman Finish for Combat Pistol, Santa's Helper Finish
+        unlock_packed_bools(42068, 42069) --Snowman Finish, Santa's Helper Finish
```
[the remark this answers is not in the document]
Comment (Vega-Punk) [reply WasabiWords.lua:168]: > Why was this removed? Snowman finish only applies to the combat pistol.

I just adapted it: [https://github.com/YimMenu-Lua/UnlockEverything/commit/e7536fb8e55f7131a14f855bc770c15d216124e4]

[the remark this answers is not in the document]
Comment (gir489returns) [reply WasabiWords.lua:168]: Fair.

File: WasabiWords.lua [hunk 5]
```diff
@@ -574,12 +629,12 @@ wasabi_words:add_button("ShinyWasabi", function() --Original script by ShinyWasa
...
-        stats.set_int('MPX_SCGW_NUM_WINS_GANG_3', 58) -- Street Crimes: Gang Wars Edition -- Win 20 games with character 4
-        stats.set_int('MPX_IAP_MAX_MOON_DIST', 2147483647) -- Invade and Persuade II -- Travel 3,474,000km on the moon
-        stats.set_int('MPX_LAST_ANIMAL', 108) -- Invade and Persuade II -- Kill 100 animals in a single playthrough
+        stats.set_int('MPX_SCGW_NUM_WINS_GANG_0', 20) -- Street Crimes: Gang Wars Edition -- Win 20 games with character 1
+        stats.set_int('MPX_SCGW_NUM_WINS_GANG_1', 20) -- Street Crimes: Gang Wars Edition -- Win 20 games with character 2
+        stats.set_int('MPX_SCGW_NUM_WINS_GANG_2', 20) -- Street Crimes: Gang Wars Edition -- Win 20 games with character 3
+        stats.set_int('MPX_SCGW_NUM_WINS_GANG_3', 20) -- Street Crimes: Gang Wars Edition -- Win 20 games with character 4
+        stats.set_int('MPX_IAP_MAX_MOON_DIST', 3474000) -- Invade and Persuade II -- Travel 3,474,000km on the moon
+        stats.set_int('MPX_LAST_ANIMAL', 100) -- Invade and Persuade II -- Kill 100 animals in a single playthrough
```
[the remark this answers is not in the document]
Comment (Vega-Punk) [reply WasabiWords.lua:637]: Why? The values ​​are correct or not?

File: WasabiWords.lua [hunk 6]
```diff
@@ -480,11 +477,69 @@ wasabi_words:add_button("ShinyWasabi", function() --Original script by ShinyWasa
...
         stats.set_int('MPX_MINIGUNS_ENEMY_KILLS', 600) -- Weapon Tints for Minigun
-        stats.set_int('MPX_SR_WEAPON_BIT_SET', -1)
-        stats.set_int('MPX_CAR_CLUB_REP', 997430)
+        stats.set_int('MPX_SR_HIGHSCORE_1', 900)
+        stats.set_int('MPX_SR_HIGHSCORE_2', 2400)
+        stats.set_int('MPX_SR_HIGHSCORE_3', 3400)
+        stats.set_int('MPX_SR_HIGHSCORE_4', 3300)
+        stats.set_int('MPX_SR_HIGHSCORE_5', 4100)
+        stats.set_int('MPX_SR_HIGHSCORE_6', 550)
+        stats.set_int('MPX_SR_WEAPON_BIT_SET', 262143)
```
[the remark this answers is not in the document]
Comment (Vega-Punk) [reply WasabiWords.lua:486]: I remove the highscore stats.

File: WasabiWords.lua (hunk 2 above)
Comment (Vega-Punk) [reply WasabiWords.lua:669]: Thanks for the info, I remove it.

File: WasabiWords.lua (hunk 5 above)
[the remark this answers is not in the document]
Comment (gir489returns) [reply WasabiWords.lua:631]: What is the reason for changing them? They were fine before.

[the remark this answers is not in the document]
Comment (Vega-Punk) [reply WasabiWords.lua:631]: It doesn't work with the correct values?? I change the old random values back.

**Pull request merged**